# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/.github/ISSUE_TEMPLATE/bug_report.md
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/.github/ISSUE_TEMPLATE/bug_report.md b/gfx/wgpu/.github/ISSUE_TEMPLATE/bug_report.md
deleted file mode 100644
--- a/gfx/wgpu/.github/ISSUE_TEMPLATE/bug_report.md
+++ /dev/null
@@ -1,28 +0,0 @@
----
-name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: bug
-assignees: ''
-
----
-
-<!-- Thank you for filing this! Please read the [debugging tips](https://github.com/gfx-rs/wgpu/wiki/Debugging-wgpu-Applications).
-That may let you investigate on your own, or provide additional information that helps us to assist.-->
-
-**Description**
-A clear and concise description of what the bug is.
-
-**Repro steps**
-Ideally, a runnable example we can check out.
-
-**Expected vs observed behavior**
-Clearly describe what you get, and how it goes across your expectations.
-
-**Extra materials**
-Screenshots to help explain your problem.
-Validation logs can be attached in case there are warnings and errors.
-Zip-compressed API traces and GPU captures can also land here.
-
-**Platform**
-Information about your OS, version of `wgpu`, your tech stack, etc.